recipes/mpir/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "2.7.2" %}
+
+package:
+  name: mpir
+  version: {{ version }}
+
+source:
+  fn: mpir-{{ version }}.tar.bz2
+  url: http://mpir.org/mpir-{{ version }}.tar.bz2
+  md5: b31aa1ebc01b5fedf4ac43277c03feec
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - m4  # [unix]
+    - python
+    - vs2015_runtime # [win]
+  run:
+    - vs2015_runtime # [win]
+
+test:
+  files:
```

Comment:
So, I was taking to @pelson about this point. We decided "no" as we don't want it to have things depend on Python that don't actually need it. That being said, in the recipe, installing Python is the easiest way to activate the correct `vc*`.